### PR TITLE
New web site navigation 

### PR DESCRIPTION
**Pull request by davidpgoo**
This PR contains updates for the website navigation and fixes for BEAM-1367, 1014, 1015

### CODE REVIEW

**Comment by kennknowles** [comment]
retest this please

**Comment by melap** [comment]
Staged: http://apache-beam-website-pull-requests.storage.googleapis.com/332/index.html

File: src/_includes/section-menu/sdks.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<li><span class="section-nav-list-main-title">SDKS</span></li>
+<li><a href="{{ site.baseurl }}/documentation/sdks/java/">Beam SDK feature comparison</a></li>
```
Comment (melap) [inline src/_includes/section-menu/sdks.html]: we should remove the SDK feature comparison link entirely for now, until it exists

**Comment by melap** [comment]
retest this please

**Pull request closed without merging**